src/main/flight/pid_init.c

```diff
@@ -224,8 +224,8 @@ void pidInitFilters(const pidProfile_t *pidProfile)
     // in-flight adjustments and transition from 0 to > 0 in flight the feature
     // won't work because the filter wasn't initialized.
     for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
-        biquadFilterInitLPF(&pidRuntime.dMinRange[axis], D_MIN_RANGE_HZ, targetPidLooptime);
-        pt1FilterInit(&pidRuntime.dMinLowpass[axis], pt1FilterGain(D_MIN_LOWPASS_HZ, pidRuntime.dT));
+        pt2FilterInit(&pidRuntime.dMinRange[axis], pt1FilterGain(D_MIN_RANGE_HZ, pidRuntime.dT));
+        pt2FilterInit(&pidRuntime.dMinLowpass[axis], pt1FilterGain(D_MIN_LOWPASS_HZ, pidRuntime.dT));
```

Comment:
For initializing it's probably not that important, but we could still change `pt1FilterGain` to `pt2FilterGain` to correctly initialize the PT2 filters.

Reply:
Thanks, fixed!

Reply:
I still see pt1FilterGain instead of pt2FilterGain.